websrc/source/documents/networking/services.md

```diff
@@ -17,16 +17,95 @@ Services offer:
 - Minimal downtime.
 
 A provider is one or more containers that match the label selector associated with the services.
+ **Note**: This feature would be deprecated since docker 1.12 onword service load balancer feature  is supported natively by docker swarm. 
 
 ## Defining Services
 
 After you create your network, you can define services using the UI, the `netctl` command line interface (CLI), or the Contiv Network REST APIs, but it is recommended
 to use the UI or API to take advantage of the authentication and authorization options in those interfaces.
 
-Note: Service requirements are defined by *selectors*. Selectors are key-value pairs 
+**Note**: Service requirements are defined by *selectors*. Selectors are key-value pairs 
 that group providers with matching labels. 
 
-To create a Service Load Balancer using the UI:
+###To create a Service Load Balancer using netctl:
+ 
+1\. Create newtwork as net0-2 in tenant default
+
+	netctl network create net0-2 -s 10.0.0.0/24 -g 10.0.0.254 -encap vxlan -t default
+2\. Create service network as svc-net-2-0 in tenant default
+
+	netctl network create svc-net-2-0 -s 30.2.0.0/24 -g 30.2.0.254 -encap vxlan -t default
+
+3\. Create service on service network as svc-0-default in tenant default
+	
+	netctl service create svc-0-default --network svc-net-2-0 -t default --selector=key0=value1 --selector=key1=value2 --selector=key2=value3 --selector=key3=value4 --port=80:8080:TCP --port=643:7070:UDP
+
+
+After above operation, check service IPs , using _netctl service inspect <service name>_. In this example, service  Ips is 30.2.0.1 
```

Comment:
Use a single backtick before and after the command to highlight it `like this`

Please remove the extraneous comma: "IPs , using" -> "IPs using"

Reply:
Using backtick will break format consistence in the document page.